apps/web/src/icons/Twittericon.tsx

```diff
@@ -1,9 +1,23 @@
-export default function TwitterIcon() {
+type Props = {
+  size?: 'sm' | 'md';
+};
+
+export default function TwitterIcon({ size = 'sm' }: Props) {
+  if (size === 'md') {
+    return (
+      <svg xmlns="http://www.w3.org/2000/svg" width="22" height="18" fill="none">
+        <path
+          fill="#000"
+          d="M21.459 2.5a8.763 8.763 0 0 1-2.471.678A4.337 4.337 0 0 0 20.88.794a8.907 8.907 0 0 1-2.736 1.036A4.3 4.3 0 0 0 10.7 4.768c0 .331.038.66.111.983a12.194 12.194 0 0 1-8.876-4.485 4.226 4.226 0 0 0-.582 2.166 4.307 4.307 0 0 0 1.914 3.584 4.292 4.292 0 0 1-1.949-.539v.053a4.306 4.306 0 0 0 3.452 4.223 4.342 4.342 0 0 1-1.935.075 4.318 4.318 0 0 0 4.028 2.99 8.629 8.629 0 0 1-5.339 1.841c-.342 0-.684-.02-1.024-.059a12.253 12.253 0 0 0 6.613 1.932A12.16 12.16 0 0 0 19.361 5.3c0-.183 0-.367-.013-.55A8.69 8.69 0 0 0 21.5 2.515l-.041-.016Z"
+        />
+      </svg>
+    );
+  }
   return (
-    <svg width="14" height="12" viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <svg xmlns="http://www.w3.org/2000/svg" width="14" height="12" fill="none">
       <path
-        d="M13.9727 1.66668C13.4487 1.89725 12.8936 2.04954 12.3253 2.11868C12.924 1.75866 13.372 1.1941 13.5867 0.529344C13.0209 0.858527 12.4046 1.0919 11.7627 1.22001C11.3699 0.79909 10.8596 0.506165 10.298 0.379378C9.73646 0.252591 9.14974 0.297815 8.61425 0.509162C8.07876 0.720509 7.61931 1.08819 7.29572 1.56432C6.97213 2.04046 6.79939 2.60299 6.8 3.17868C6.80021 3.39919 6.82503 3.619 6.874 3.83401C5.73296 3.77936 4.61625 3.48484 3.59667 2.96965C2.5771 2.45446 1.67755 1.73018 0.956667 0.844011C0.699636 1.28161 0.565581 1.78052 0.568667 2.28801C0.568341 2.76052 0.684628 3.2258 0.907214 3.6426C1.1298 4.05939 1.4518 4.41482 1.84467 4.67734C1.38912 4.66276 0.943644 4.53956 0.545333 4.31801V4.35334C0.544943 5.01615 0.773923 5.65867 1.19341 6.17185C1.61289 6.68502 2.19703 7.03722 2.84667 7.16868C2.42591 7.28224 1.98495 7.29933 1.55667 7.21868C1.74107 7.78907 2.09893 8.28767 2.58033 8.6449C3.06173 9.00212 3.64264 9.20015 4.242 9.21134C3.2269 10.0084 1.97327 10.4409 0.682667 10.4393C0.45456 10.4388 0.226656 10.4257 0 10.4C1.31614 11.2421 2.84618 11.6891 4.40867 11.688C5.48225 11.6959 6.54671 11.4904 7.54023 11.0835C8.53375 10.6766 9.43655 10.0764 10.1962 9.31778C10.9559 8.55912 11.5572 7.65711 11.9654 6.66412C12.3736 5.67112 12.5805 4.60694 12.574 3.53334C12.574 3.41134 12.574 3.28868 12.5653 3.16601C13.1289 2.75982 13.6149 2.25557 14 1.67734L13.9727 1.66668Z"
-        fill="black"
+        fill="#000"
```

Comment:
Open for suggestion on how do we handle icon sizing here 🤔 

Didn't put in the `<IconContainer/>` since its not clickable and no need the hover state

Reply:
I think if we don't specify the width and height inline in the svg, we can resize it based on the container where its used